Add R3F v9 in peerDependencies

Installing R3F v9 with Drei gives the following error due to the [@react-three/fiber@">=8.0 peerDependency](https://github.com/pmndrs/drei/blob/v10/package.json#L148):

```
➜  next15-app-r3f9-react19 git:(main) ✗ npm i
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: next15-app-r3f9-react19@0.1.0
npm error Found: @react-three/fiber@9.0.0-rc.1
npm error node_modules/@react-three/fiber
npm error   @react-three/fiber@"9.0.0-rc.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer @react-three/fiber@">=8.0" from @react-three/drei@9.120.0
npm error node_modules/@react-three/drei
npm error   @react-three/drei@"9.120.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
```

With the following dependencies:

```json
  "dependencies": {
    "@react-three/drei": "9.120.0",
    "@react-three/fiber": "9.0.0-rc.1",
    "react": "19.0.0",
    "react-dom": "19.0.0",
    "next": "15.0.4",
    "three": "0.171.0"
  }
```

- Node v22.10.0
- NPM v10.9.0

In the meantime, `npm i --legacy-peer-deps` works.

--- a/package.json
+++ b/package.json
@@ -131,7 +131,7 @@
     "vite-plugin-glslify": "^2.1.0"
   },
   "peerDependencies": {
-    "@react-three/fiber": "^8",
+    "@react-three/fiber": "^8 || ^9.0.0-0",
     "react": "^18",
     "react-dom": "^18",
     "three": ">=0.137"

